src/core/biblio-db.js

```diff
@@ -19,29 +19,27 @@ const readyPromise = openDB("respec-biblio2", 12, {
     Array.from(db.objectStoreNames).map(storeName =>
       db.deleteObjectStore(storeName)
     );
-    const promisesToCreateSchema = [
-      new Promise((resolve, reject) => {
-        try {
-          const store = db.createObjectStore("alias", { keyPath: "id" });
-          store.createIndex("aliasOf", "aliasOf", { unique: false });
-          store.transaction.oncomplete = resolve;
-          store.transaction.onerror = reject;
-        } catch (err) {
-          reject(err);
-        }
-      }),
-      new Promise((resolve, reject) => {
-        try {
-          const transaction = db.createObjectStore("reference", {
-            keyPath: "id",
-          }).transaction;
-          transaction.oncomplete = resolve;
-          transaction.onerror = reject;
-        } catch (err) {
-          reject(err);
-        }
-      }),
-    ];
+    new Promise((resolve, reject) => {
+      try {
+        const store = db.createObjectStore("alias", { keyPath: "id" });
+        store.createIndex("aliasOf", "aliasOf", { unique: false });
+        store.transaction.oncomplete = resolve;
+        store.transaction.onerror = reject;
+      } catch (err) {
+        reject(err);
+      }
+    });
+    new Promise((resolve, reject) => {
+      try {
+        const transaction = db.createObjectStore("reference", {
+          keyPath: "id",
+        }).transaction;
+        transaction.oncomplete = resolve;
+        transaction.onerror = reject;
+      } catch (err) {
+        reject(err);
+      }
+    });
```

Comment:
Can we just skip the promise part here? The [idb examples](https://github.com/jakearchibald/idb#article-store) or even the raw [IndexedDB examples](https://developer.mozilla.org/en-US/docs/Web/API/IDBObjectStore/createIndex#Example) never uses `transaction.oncomplete` so maybe?

Reply:
I will look into it and other parts of the module. Actually while refactoring I only changed parts for which `idb` returns a promise. Since you wanted this issue fixed for your work, I made a PR as soon as I got the tests resolved. I will make these changes.

Reply:
I was thinking the same. Hopefully idb takes care of everything and the above goes away.